insights/parsers/netstat.py

```diff
@@ -292,17 +297,14 @@ def add_data(self, line):
 
     def _merge_data_index(self):
         merged_data = {}
-        component_len = {}
         i = 0
         while i < len(self.indexes):
             index = self.indexes[i]
             m = self.meta[index]
             merged_data[m] = self.data[i]
-            component_len[m] = self.indexes[i + 1] - self.indexes[i] if i != len(self.indexes) - 1 else None
             i += 1
 
         self.data = merged_data
-        self.data[COMPONENT_LEN] = component_len
```

Comment:
@PaulWay since this is in the existing api, even thought it does not appear to have been tested, we should leave this in unless there's a problem associated with it.

Reply:
That seems to be unnecessary work to me, but OK.